src/poprox_recommender/components/joiners/rrf.py

```diff
@@ -0,0 +1,41 @@
+from collections import defaultdict
+
+from poprox_concepts import ArticleSet
+from poprox_recommender.lkpipeline import Component
+from poprox_recommender.lkpipeline.types import Lazy
+
+
+class ReciprocalRankFusion(Component):
+    def __init__(self, num_slots: int):
+        self.num_slots = num_slots
+
+    def __call__(self, candidates1: ArticleSet, candidates2: Lazy[ArticleSet]) -> ArticleSet:
```

Comment:
Since RRF always uses `candidates2`, it does not need to be `Lazy` (and should not be).